src/redux/slices/user.ts

```diff
@@ -35,7 +34,6 @@ import {
   DEFAULT_TRANSACTION,
   formatContact,
   formatContactForController,
-  formatTransaction,
```

Comment:
if not used, remove the function from utils too

Reply:
my god, didn't do that XDDDDDDD